runtime/common/src/weights/parachain_staking.rs

```diff
@@ -50,64 +50,64 @@ pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> parachain_staking::weights::WeightInfo for WeightInfo<T> {
     // Storage: ParachainStaking DelegatorState (r:1 w:1)
     fn hotfix_remove_delegation_requests(x: u32) -> Weight {
-        (7_627_000 as Weight)
-            // Standard Error: 42_000
-            .saturating_add((13_072_000 as Weight).saturating_mul(x as Weight))
+        (0 as Weight)
+            // Standard Error: 75_000
+            .saturating_add((14_571_000 as Weight).saturating_mul(x as Weight))
             .saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(x as Weight)))
             .saturating_add(T::DbWeight::get().writes((1 as Weight).saturating_mul(x as Weight)))
     }
```

Comment:
This has weight zero if `x` is zero. `hotfix_remove_delegation_requests` is just an iteration over an empty iterator if `x` is zero, so I guess that's fine: https://github.com/PureStake/moonbeam/commit/845816af288eb28f11fd9fa9d058fee903d84e8f

(The moonbeam benchmarks also have zero base weight.)

Reply:
https://github.com/PureStake/moonbeam/blob/845816af288eb28f11fd9fa9d058fee903d84e8f/pallets/parachain-staking/src/lib.rs#L1784-L1792

It's also `ensure_root`. I think it's fine too 👍. But good attention.